packages/console/src/components/Dropdown/index.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { ReactNode, RefObject, useEffect, useRef } from 'react';
+import ReactModal from 'react-modal';
+
+import * as styles from './index.module.scss';
+import usePosition from './use-position';
+
+export { default as DropdownItem } from './DropdownItem';
+
+type Props = {
+  children: ReactNode;
+  title?: ReactNode;
+  isOpen: boolean;
+  onClose?: () => void;
+  anchorRef: RefObject<HTMLElement>;
+  isFullWidth?: boolean;
+};
+
+const Dropdown = ({ children, title, isOpen, onClose, anchorRef, isFullWidth }: Props) => {
+  const overlayRef = useRef<HTMLDivElement>(null);
+
+  const { position, mutate } = usePosition(anchorRef, overlayRef);
+
+  useEffect(() => {
+    if (isOpen) {
+      // Calc position after modal opened.
+      setTimeout(() => {
+        mutate();
+      });
```

Comment:
Could move to `ReactModal`'s  onAfterOpen callback params?

Reply:
updated